chapters/en/slides/module5/module5_03.Rmd

```diff
@@ -0,0 +1,316 @@
+is that it is not limited to static visualizations.
+
+By simply appending the `interactive()` method to our chart 
+we can use the mouse to zoom by scrolling and pan by clicking and dragging!
+
+This allows us to interactively explore the data
+and zoom in on the points of interest as needed.
+We will learn more about interactive visualizations in the last module,
+this is just a sample of what is possible.
+
+How can we solve the problem of a large axis range in a static figure?
+
+---
+
+## A log transformed axis can reveal detail across a large range of values
+
+```{python}
+alt.Chart(donations).mark_point().encode(
+    alt.X('date'),
+    alt.Y('sum', axis=alt.Axis(format='$'), scale=alt.Scale(type='log')),
+    alt.Tooltip('week_day'))
+```
+
+Notes:
+By applying a logarithmic (log) transformation to the y-axis,
+we're able to increase the range used for the smaller values
+by compressing the space used for larger values.
+
+This means that log transformations are useful
+when there are large differences in the data values,
+such as in our case here.
+
+Log scales are also useful to show data that has been obtained from multiplication or division,
+such as ratios.
+
+The default log transformaton in Altair uses the base 10
+which is the most common choice.
+Every step on a log scale is referred to
+as a difference of an "order of magnitude".
+
+It is important to remember that we log scaled axes
+are not as easy to interpret as linear ones,
+and it is easy to underestimate actual the difference between large and small values.
+
+This is especially true
+when creating figures for communication
+with the general public
+rather than in a subject area
+where participants might be more used
+to seeing this transformation.
+
+---
+
+## The log transformation does not work for data containing the value zero
+
+```{python}
+donations.loc[4461, 'sum'] = 0
+alt.Chart(donations).mark_point().encode(
+    alt.X('date'),
+    alt.Y('sum', axis=alt.Axis(format='$'), scale=alt.Scale(type='log')))
+```
+
+Notes:
+The way the logarithmic transform is defined
+means that it cannot be calculated for the value 0.
+
+We can see this is we artificially introduce a 0 in our dataset.
+
+All the datapoints are compressed at the top,
+because the log of 0 is interpretes as a value of negative infinity.
+
+---
+
+## The symlog transformation work around the problem with zeroes in the data
+
+```{python}
+alt.Chart(donations).mark_point().encode(
+    alt.X('date'),
+    alt.Y('sum', axis=alt.Axis(format='$'), scale=alt.Scale(type='symlog')))
+```
+
+Notes:
+Fortunately there is a symmetrical variant of the log transformation (symlog)
+which works around the issues with zeroes by definining a linear range for values close to 0,
+and a logarithmic range above and beyond these valules.
+
+We can see that the y-axis still looks a bit cryptic when using this transformation,
+but it allows us to plot the values sucessfully
+even when they contain zero.
```

Comment:
This is not clear, please explain a bit more - in particular, the y-axis looking cryptic.